src/website/app/demos/Pagination/examples/basic.js

```diff
@@ -0,0 +1,46 @@
+/* @flow */
+import { Component } from 'react';
+import Pagination from '../../../../../library/Pagination';
+
+export default {
+  id: 'basic',
+  title: 'Basic Usage',
+  description: `Pass Pagination a \`currentPage\` to render, a \`pageSize\`
+(number of items or data to be rendered), an \`onPageChange\` handler, and the
```

Comment:
This paragraph is hard to read.  Suggest to remove "items or data", "data or items" and just use "items".